src/wp-includes/class-wp-term.php

```diff
@@ -195,7 +198,14 @@ public static function get_instance( $term_id, $taxonomy = null ) {
 	 * @param WP_Term|object $term Term object.
 	 */
 	public function __construct( $term ) {
+		$int_fields = array( 'parent', 'term_id', 'count', 'term_group', 'term_taxonomy_id', 'object_id' );
 		foreach ( get_object_vars( $term ) as $key => $value ) {
+			if ( in_array( $key, $int_fields, true ) ) {
```

Comment:
```suggestion
			if ( isset( $int_fields[ $key ] ) ) {
```
Use `isset` instead `in_array`

Reply:
This doesn't work. Isset is used for checking if a key existings. `$int_fields` is a numberic array, this does not work.